core/src/test/java/org/jdbi/v3/core/argument/TestPreparedArguments.java

```diff
@@ -22,7 +22,7 @@
 
 public class TestPreparedArguments {
     @RegisterExtension
-    public H2DatabaseExtension h2Extension = H2DatabaseExtension.withPlugins();
```

Comment:
this change needs to make sure that there is no plugin actually loaded from the classpath. 

I have no problem deprecating (and removing a some point!) the withPlugins() and installPlugins() methods; they always felt wrong to me (and here it seems to bite us).

Reply:
The test does not rely on any plugins being present at all.